plugins/plugins/frontend/public/javascripts/countly.views.js

```diff
@@ -1731,5 +1731,15 @@ $(document).ready(function() {
             app.addMenu("management", {code: "plugins", url: "#/manage/plugins", text: "plugins.title", icon: '<div class="logo-icon fa fa-puzzle-piece"></div>', priority: 30});
         }
         app.addMenu("management", {code: "configurations", url: "#/manage/configurations", text: "plugins.configs", icon: '<div class="logo-icon ion-android-options"></div>', priority: 40});
+
+        var isCurrentHostnameIP = /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/.test(window.location.hostname);
+        if (!isCurrentHostnameIP && countlyGlobal.domain === "") {
+            countlyPlugins.updateConfigs({"api": {"domain": window.location.hostname}}, function(err) {
+                if (err) {
+                    // throw err
+                    console.log('An error has occurred: ', err);
```

Comment:
Eslint was complaining about this line.
Normally we do it like that:
1)If error happens after user action - we show styled notification.
2)If it happens in background - then we just ignore it silently. (So do not log it in console)